src/auth/iam.ts

```diff
@@ -40,6 +42,7 @@ export class IAMAuth {
    */
   createScopedRequired() {
```

Comment:
You missed the `@deprecated` annotation in the comment. Since you added it elsewhere, add it here too.